boards/DCM/Src/App/App_SetPeriodicCanSignals.c

```diff
@@ -0,0 +1,91 @@
+#include "App_SharedSetPeriodicCanSignals.h"
+#include "App_SetPeriodicCanSignals.h"
+#include "App_InRangeCheck.h"
+
+STATIC_DEFINE_APP_SET_PERIODIC_CAN_SIGNALS_IN_RANGE_CHECK(DcmCanTxInterface)
+
+// TODO: Implement PID controller to maintain DC bus power at 80kW
+// #680
+void App_SetPeriodicCanSignals_TorqueRequests(const struct DcmWorld *world)
+{
+    struct DcmCanRxInterface *can_rx = App_DcmWorld_GetCanRx(world);
+    struct DcmCanTxInterface *can_tx = App_DcmWorld_GetCanTx(world);
+
+    const float MAX_SAFE_TORQUE_REQUEST_NM =
+        21.0f; // 21Nm is max torque each motor can handle
+
+    // Regen allowed when braking or (speed > 5kmh and AIRs closed)
+    float regen_allowed_threshold_kph = 5.0f;
+    bool  is_every_air_closed =
+        (App_CanRx_BMS_AIR_STATES_GetSignal_AIR_POSITIVE(can_rx) ==
+         CANMSGS_BMS_AIR_STATES_AIR_POSITIVE_CLOSED_CHOICE) &&
+        (App_CanRx_BMS_AIR_STATES_GetSignal_AIR_NEGATIVE(can_rx) ==
+         CANMSGS_BMS_AIR_STATES_AIR_NEGATIVE_CLOSED_CHOICE);
+    bool is_vehicle_above_regen_allowed_threshold =
+        (App_CanRx_FSM_WHEEL_SPEED_SENSOR_GetSignal_LEFT_WHEEL_SPEED(can_rx) >
+         regen_allowed_threshold_kph) &&
+        (App_CanRx_FSM_WHEEL_SPEED_SENSOR_GetSignal_RIGHT_WHEEL_SPEED(can_rx) >
+         regen_allowed_threshold_kph);
+    bool is_regen_allowed = is_vehicle_above_regen_allowed_threshold && is_every_air_closed;
+
+    float regen_paddle_percentage =
+        (float)App_CanRx_DIM_REGEN_PADDLE_GetSignal_MAPPED_PADDLE_POSITION(
+            can_rx);
+    float torque_request;
+
+    if (regen_paddle_percentage > 0 && is_regen_allowed)
+    {
+        torque_request =
```

Comment:
I'd leave a comment for why the torque request is negative here

Reply:
I'd also include an equation showing the reader how to calculate the torque request. Something like this would be nice 
![image](https://user-images.githubusercontent.com/32230980/99202833-5c616900-2765-11eb-9d92-3c954f38bb96.png)
(taken from `Io_CurrentSense.c`)

Reply:
ASCII art time